README.md

```diff
@@ -474,82 +471,88 @@ Given a value `f`, one can access its type representative via the
+
+<a name="alt-method"></a>
 
-#### `alt` method
+#### `fantasy-land/alt` method
 
 ```hs
-alt :: Alt f => f a ~> f a -> f a
+fantasy-land/alt :: Alt f => f a ~> f a -> f a
 ```
 
-A value which has a Alt must provide a `alt` method. The
-`alt` method takes one argument:
+A value which has a Alt must provide a `fantasy-land/alt` method. The
+`fantasy-land/alt` method takes one argument:
 
-    a.alt(b)
+    a['fantasy-land/alt'](b)
 
 1. `b` must be a value of the same Alt
 
-    1. If `b` is not the same Alt, behaviour of `alt` is
+    1. If `b` is not the same Alt, behaviour of `fantasy-land/alt` is
        unspecified.
     2. `a` and `b` can contain any value of same type.
     3. No parts of `a`'s and `b`'s containing value should be checked.
 
-2. `alt` must return a value of the same Alt.
+2. `fantasy-land/alt` must return a value of the same Alt.
 
 ### Plus
 
 A value that implements the Plus specification must also implement
 the [Alt](#alt) specification.
 
-1. `x.alt(A.zero())` is equivalent to `x` (right identity)
-2. `A.zero().alt(x)` is equivalent to `x` (left identity)
-3. `A.zero().map(f)` is equivalent to `A.zero()` (annihilation)
+1. `x['fantasy-land/alt'](A['fantasy-land/zero']())` is equivalent to `x` (right identity)
+2. `A['fantasy-land/zero']()['fantasy-land/alt'](x)` is equivalent to `x` (left identity)
+3. `A['fantasy-land/zero']()['fantasy-land/map'](f)` is equivalent to `A['fantasy-land/zero']()` (annihilation)
 
-#### `zero` method
+<a name="zero-method"></a>
+
+#### `fantasy-land/zero` method
 
 ```hs
-zero :: Plus f => () -> f a
+fantasy-land/zero :: Plus f => () -> f a
 ```
 
-A value which has a Plus must provide an `zero` function on its
+A value which has a Plus must provide a `fantasy-land/zero` function on its
 [type representative](#type-representatives):
 
-    A.zero()
+    A['fantasy-land/zero']()
 
 Given a value `x`, one can access its type representative via the
 `constructor` property:
 
-    x.constructor.zero()
+    x.constructor['fantasy-land/zero']()
 
-1. `zero` must return a value of the same Plus
+1. `fantasy-land/zero` must return a value of the same Plus
 
 ### Alternative
 
 A value that implements the Alternative specification must also implement
 the [Applicative](#applicative) and [Plus](#plus) specifications.
 
-1. `x.ap(f.alt(g))` is equivalent to `x.ap(f).alt(x.ap(g))` (distributivity)
-2. `x.ap(A.zero())` is equivalent to `A.zero()` (annihilation)
+1. `x['fantasy-land/ap'](f['fantasy-land/alt'](g))` is equivalent to `x['fantasy-land/ap'](f)['fantasy-land/alt'](x['fantasy-land/ap'](g))` (distributivity)
+2. `x['fantasy-land/ap'](A['fantasy-land/zero']())` is equivalent to `A['fantasy-land/zero']()` (annihilation)
 
 ### Foldable
 
-1. `u.reduce` is equivalent to `u.reduce((acc, x) => acc.concat([x]), []).reduce`
+1. `u['fantasy-land/reduce']` is equivalent to `u['fantasy-land/reduce']((acc, x) => acc.concat([x]), []).reduce`
```

Comment:
We really do want `.concat` and `.reduce` here because we're referring to `Array#concat` and `Array#reduce`.